frontend/src/global_styles/openproject/_mixins.sass

```diff
@@ -81,9 +81,9 @@
     text-decoration: none
 
 @mixin widget-box--style
-  background: var(--body-background)
+  background: var(--overlay-bgColor)
```

Comment:
That looks also semantically incorrect. Github has widget boxes as well. e.g for pinned repositories. Please double check what they use there